geode-pulse/src/main/webapp/WEB-INF/spring-security.xml

```diff
@@ -74,13 +74,17 @@
 	
 	<beans:bean id="gemAuthenticationProvider" 
 		class="org.apache.geode.tools.pulse.internal.security.GemFireAuthenticationProvider">
-  	</beans:bean>
+  </beans:bean>
 
+	<beans:bean id="passwordEncoder"
```

Comment:
this is only needed for Pulse? the other webapps don't need it?

Reply:
No, because Pulse uses a static file by default (containing the plaintext passwords)